src/cascadia/TerminalSettingsEditor/MainPage.idl

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.Terminal.Settings.Editor
     runtimeclass MainPage : Windows.UI.Xaml.Controls.Page
     {
         MainPage(Microsoft.Terminal.Settings.Model.CascadiaSettings settings);
+
+        Microsoft.Terminal.Settings.Model.CascadiaSettings Settings { get; };
```

Comment:
```suggestion
        static Microsoft.Terminal.Settings.Model.CascadiaSettings Settings { get; };
```

Reply:
I don’t necessarily recommend making this static?????